features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/default/templates/sso/filter/template.jag

```diff
@@ -32,10 +32,14 @@
             if(site.ssoConfiguration.keyStoreName == null || site.ssoConfiguration.keyStoreName == ""){
                 site.ssoConfiguration.keyStoreName = CarbonUtils.getServerConfiguration().getFirstProperty("Security.KeyStore.Location");
                 }
+            if(site.ssoConfiguration.nameIdPolicy == null){
```

Comment:
Fixed in #1581(https://github.com/wso2/carbon-apimgt/commit/ba3007eb347b47f68e1dd6e75fe8102d057200dc)